.github/workflows/dev-release.yml

```diff
@@ -37,5 +40,5 @@ jobs:
           npm run styles && \
           cd `echo ${ELEMENT_DIR} | tr -d '[]"'` && \
           npm run build && \
-          npm version prerelease --preid ${{ inputs.preid }} && \
-          npm publish --access=public --tag ${{ inputs.preid }}
+          npm version prerelease --preid ${{ inputs.preid }}.${{ steps.timestamp.outputs.timestamp }} && \
+          npm publish --access=public --tag ${{ inputs.preid }}.${{ steps.timestamp.outputs.timestamp }}
```

Comment:
Thanks! I guess in the tag we don't need it, as the tag should ideally always be `dev` (or something else, like `beta`, or `pr` to indicate that it comes from a PR etc.)

See example from openlayers:
![image](https://github.com/user-attachments/assets/17fe29fb-4253-4157-80c8-363c0fb8c59a)